themes/repair-cafe/partials/components/eventlist.htm

```diff
@@ -5,40 +5,29 @@
             <ul class="events list-unstyled mb-5">
                 {% for event in events %}
                     {% set static_map_url = event.getStaticMapURL %}
-                    {% set has_additional_information = (static_map_url and event.description is not empty) %}
```

Comment:
removing this means, that the expand-/collapse-arrow is always shown? if this is the intention it seems okay to me.